lib/Yield/PTML.cpp

```diff
@@ -108,6 +108,18 @@ static std::string labelDefinition(const BasicBlockID &BasicBlock,
              .serialize();
 }
 
+static std::string labelReference(const BasicBlockID &BasicBlock,
+                                  const yield::Function &Function,
+                                  const model::Binary &Binary) {
```

Comment:
Ah! I forgot to fix the commit name.

When running `check-conventions` at the very end, the previous commit wasn't compiling because of "unused static function", so I just moved it down one. My bad for forgetting I explicitly mentioned it by name 😅.

Reply:
Fixed.